Unity-Technologies/ScriptableRenderPipeline
# Fixed resource lifecyle issue.

### Purpose of this PR
 - Fixed resource lifetime issue that caused references to be lost in case of multiple renders assigned in the pipeline asset.
 - Fixed a compiler issue in VR when having URP without VR module.

---
### Testing status

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- [x ] Checked new UI names with UX convention
- [x ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [x ] C# and shader warnings (supress shader cache to see them)
- [x ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

---
### Comments to reviewers
 Changelog is not required as this is fixing a regression in a package not released yet. (only in 2020.1)
 This will allow to backport another PR that fixed resource leak in post-processing to 2019.3
